druid/src/command.rs

```diff
@@ -34,17 +34,8 @@ pub struct Selector(&'static str);
 /// A `Command` consists of a [`Selector`], that indicates what the command is,
 /// and an optional argument, that can be used to pass arbitrary data.
 ///
-///
-/// # One-shot and reusable `Commands`
-///
-/// Commands come in two varieties, 'reusable' and 'one-shot'.
-///
-/// Regular commands are created with [`Command::new`], and their argument
-/// objects may be accessed repeatedly, via [`Command::get_object`].
-///
-/// One-shot commands are intended for cases where an object should only be
-/// used once; an example would be if you have some resource that cannot be
-/// cloned, and you wish to send it to another widget.
+/// If the payload can't or shouldn't be cloned,
+/// wrapping it with [`SingleUse`] allows `take`ing the object.
```

Comment:
Given that *takeing* is a misspelling, perhaps write it as: *allows you to `take` the object.*